docs-gen/includes/quickstart/resources/runconfiguration.yaml

```diff
@@ -8,4 +8,6 @@ spec:
       experimentName: penguin-experiment
     triggers:
       schedules:
-      - '0 * * * *'
+      - cronExpression: '0 * * * *'
+        startTime: "2024-01-01T00:00:00Z"
+        endTime: "2024-12-31T23:59:59Z"
```

Comment:
stick to single or double quotes?

Reply:
a few other places

Reply:
we didn't need the quotes at all, so removed them in 1060cf492503107798674c9aa4752556a45ed919